katsdpimager/clean.py

```diff
@@ -875,8 +875,8 @@ def __call__(self, psf_patch, threshold=0.0):
             self.command_queue.finish()
         peak_value = peak_value_device.get_async(self.command_queue, self._peak_value_host)
         peak_pos = peak_pos_device.get_async(self.command_queue, self._peak_pos_host)
-        peak_pos = tuple(int(x) for x in peak_pos)
         self.command_queue.finish()
```

Comment:
So this is like an `await` for all the `get_async`s before it?

Reply:
Exactly.